.github/workflows/pluto_build.yml

```diff
@@ -73,26 +74,22 @@ jobs:
 
       - name: Set build environment variables
         run: |
-          # replace dashes with underscores to create a valid postgres schema name
-          BUILD_ENGINE_SCHEMA=$(echo ${BUILD_ENGINE_SCHEMA} | tr - _)
-          echo "BUILD_ENGINE_SCHEMA=$BUILD_ENGINE_SCHEMA" >> "$GITHUB_ENV"
-          # set postgres schema search path to prioritize BUILD_ENGINE_SCHEMA
-          BUILD_ENGINE=${BUILD_ENGINE_DATABASE}?options=--search_path%3D${BUILD_ENGINE_SCHEMA},public
+          BUILD_ENGINE=${BUILD_ENGINE_SERVER}/${BUILD_ENGINE_DB}
           echo "BUILD_ENGINE=$BUILD_ENGINE" >> "$GITHUB_ENV"
 
-      - name: Plan Build
-        working-directory: ./
-        run: |
-          python3 -m dcpy.builds.load pluto
-
       - name: Set versions env vars
         working-directory: products/pluto/
         run: |
           echo VERSION=$(yq .version recipe.lock.yml) >> "$GITHUB_ENV" && \
           echo VERSION_PREV=$(yq '.inputs.datasets.[] | select(.name == "dcp_pluto") | .version' recipe.lock.yml) >> "$GITHUB_ENV"
 
-      - name: Data setup
-        run: ./00_setup.sh
+      # - name: Data setup
+      #   run: ./00_setup.sh
+
+      - name: Load data
+        working-directory: ./
```

Comment:
I see this was done before, but I think this should be run from the pluto directory? dcpy is installed via pip prior to this, so we don't need to use the local filesystem for it. Doesn't really make a functional difference, but it'd be consistent if in the future we're directly invoking python from the build directory.

Unless there's still code in dcpy which has logic based on relative directory

Reply:
good call, see [83afce4](https://github.com/NYCPlanning/data-engineering/pull/291/commits/83afce469e0083e946ebd19465194617b2fb54f3)